projects/client/RabbitMQ.Client/src/client/api/ConnectionFactory.cs

```diff
@@ -488,13 +500,13 @@ public IFrameHandler CreateFrameHandlerForHostname(string hostname)
         private IFrameHandler ConfigureFrameHandler(IFrameHandler fh)
         {
             // make sure socket timeouts are higher than heartbeat
-            fh.ReadTimeout  = Math.Max(SocketReadTimeout,  RequestedHeartbeat * 1000);
+            fh.ReadTimeout = Math.Max(SocketReadTimeout, RequestedHeartbeat * 1000);
             fh.WriteTimeout = Math.Max(SocketWriteTimeout, RequestedHeartbeat * 1000);
             // TODO: add user-provided configurator, like in the Java client
             return fh;
         }
 
-        public void SetUri(Uri uri)
+        private void SetUri(Uri uri)
```

Comment:
Yes, `master` will be 5.0 (`5.0.1`, to be specific, because an accidental release a few weeks ago).